lib/msal-angular/src/msal.guard.ts

```diff
@@ -14,7 +14,7 @@ import { concatMap, catchError, map } from "rxjs/operators";
 import { Observable, of } from "rxjs";
 
 @Injectable()
-export class MsalGuard implements CanActivate {
+export class MsalGuard implements CanActivate, CanActivateChild {
```

Comment:
Is this broker related?

Reply:
No, this is related to child routes in Angular (e.g. contoso.com/product/123)

Reply:
Nope! This is in response to an Angular issue from 1.x. I don't think it will affect broker.